acme/acme.go

```diff
@@ -344,7 +344,9 @@ Let's Encrypt functionality will be limited until traefik is restarted.`, err)
 		log.Info("Register...")
 		reg, err := a.client.Register()
 		if err != nil {
-			return err
+			log.Errorf(`Failed to register user: %s
+Let's Encrypt functionality will be limited until traefik is restarted.`, err)
```

Comment:
Shouldn't Traefik be capitalized?

Reply:
Maybe: "You will need to restart Traefik to resume normal operation with Let's Encrypt" more accurate?